docs/reference/faqs.md

```diff
@@ -75,6 +75,16 @@ The helm chart supports instantiating both consoles and engines.  Ingress to the
 Yes, just like Docker, you will be able to use Podman for container orchestration.
 </details>
 
+<details>
+  <summary>Why does Ping recommand K8s vs docker?</summary>
+
+<br>&emsp;1. Docker or a pure container solution like ECS by itself is generally not as robust or resilient as a K8s environment.  That being said, managed Docker services like ECS can provide some of the kubernetes trappings, but that locks you into AWS only and you would have a slightly different experience at Google, Azure, etc.  Kubernetes, even managed like EKS, provide more flexibility and portability</br>
+<br>&emsp;2. It is the model we use for our SaaS offerings, so support, sales, etc. are more familiar with that model</br>
```

Comment:
It is the model we use for our SaaS offerings, so internal teams at Ping are more familiar with this model.</br>